game-core/src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -106,12 +106,36 @@ private Matches() {}
     return unit -> UnitAttachment.get(unit.getType()).getIsSea();
   }
 
-  public static Predicate<Unit> unitIsSub() {
-    return unit -> UnitAttachment.get(unit.getType()).getIsSub();
+  // public static Predicate<Unit> unitIsSub() {
+  // return unit -> UnitAttachment.get(unit.getType()).getIsSub();
+  // }
```

Comment:
This commented out code should be removed

Reply:
Yep. Removed.